dask/dataframe/core.py

```diff
@@ -577,6 +577,40 @@ def _aca_agg(self, token, func, aggfunc=None):
         """ Wrapper for aggregations """
         raise NotImplementedError
 
+    def rolling(self, window, min_periods=None, win_type=None):
+        """Provides rolling transformations.
+
+        Parameters
+        ----------
+
+        window: int
+            Size of the moving window. This window must be smaller
+            than the size of the previous partition.
+        min_periods: int, default None
+            Minimum number of observations in window required to have
+            a value (otherwise result is NA).
+        win_type: string, default None
+            Provide a window type. (Identical to pandas.)
+
+        The center, freq, and axis arguments are not supported.
+        """
+        from dask.dataframe.rolling import Rolling
+
+        if not isinstance(window, int):
+            raise TypeError('window must be an integer')
+        if window < 0:
+            raise ValueError('window must be positive')
+
+        if min_periods is not None:
+            if not isinstance(min_periods, int):
+                raise TypeError('min_periods must be an integer')
+            if min_periods < 0:
+                raise ValueError('min_periods must be positive')
+
+        return Rolling(self, {'window': window,
```

Comment:
This is an api change without a deprecation of the old api (may break user code). For the other pandas api change such as `resample` we implemented deprecation logic based on pandas version (see https://github.com/dask/dask/blob/master/dask/dataframe/tseries/resample.py#L19) - perhaps the same should be done here?